src/pages/iou/steps/IOUParticipantsPage.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {
+    View,
+    Text,
+    TouchableOpacity,
+    ActivityIndicator,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../../styles/styles';
+import themeColors from '../../../styles/themes/default';
+
+const propTypes = {
+    // callback to inform parent modal of success
+    onStepComplete: PropTypes.func.isRequired,
+
+    // is this IOU request for a group bill split
+    hasMultipleParticipants: PropTypes.bool.isRequired,
+
+    // is page content being retrieved
+    isLoading: PropTypes.bool.isRequired,
+};
+
+const IOUParticipantsPage = props => (
+    <View style={styles.settingsWrapper}>
+        <Text style={[styles.buttonText]}>
+            {props.hasMultipleParticipants ? '// group' : '// single'}
```

Comment:
OK, here is where I wanted to make the suggestion on this... Rather than passing this as a prop, since it's ALREADY part of the URL, I think that you should use a `<Route>` with two different paths in order to determine if there are multiple participants or not. Like so:

```
<Route path="iou/request">
    // single
</Route>
<Route path="iou/split">
    // group
</Route>
```

This will also clean up the main Router paths because there will only be a single route to match for the `IOUModal` (and clarifies some confusion with one component being used for two different routes).

Reply:
Nice, this is much better. I had to make the RightDockedModal take an array of routes instead of a single String, but i think I got it. [Link to commit](https://github.com/Expensify/Expensify.cash/pull/1561/commits/bc15adbaf036e5949c4c922955c390912b102bc1)